src/EnergyPlus/HVACFourPipeBeam.hh

```diff
@@ -0,0 +1,280 @@
+#ifndef HVACFourPipeBeam_hh_INCLUDED
+#define HVACFourPipeBeam_hh_INCLUDED
+
+#include <memory>
+// ObjexxFCL Headers
+#include <ObjexxFCL/Array1.hh>
+
+// EnergyPlus Headers
+#include <AirTerminalUnit.hh>
+#include <EnergyPlus.hh>
+#include <DataGlobals.hh>
+#include <GlobalNames.hh>
+#include <PlantLocation.hh> // this is from https://github.com/NREL/EnergyPlus/pull/4945  assuming it goes in other wise maybe the same from using DataPlant
+
+namespace EnergyPlus {
+
+namespace FourPipeBeam {
+
+class HVACFourPipeBeam : public AirTerminalUnit
+{
+
+public: // Creation
+
+	// Default Constructor
+	HVACFourPipeBeam() :
```

Comment:
@EnergyArchmage If you have a static Factory method to create this object, why is the constructor public? Should it be private (or protected)? It should work to be private but I would have to double check to see if it compiles for sure.